### ISSUE TITLE
Learning is zero for the first iteration when warmup enabled.

### ISSUE THREAD

**Issue by engint-cerebras** [volcengine/verl#3674]
### System Info

Constant LR with Linear warmup sets LR to 0 for the first weight update since `self.actor_lr_scheduler.step` called after the `self.actor.update_policy` in `ActorRolloutRefWorker.update_actor`.

https://github.com/volcengine/verl/blob/3eaaf24d5a2c7bd0653fdf9b791d54ec3d93a665/verl/workers/fsdp_workers.py#L590

### Information

- [x] The official example scripts
- [ ] My own modified scripts

### Tasks

- [x] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

Any valid script should reproduce this.

### Expected behavior

I don't know. Maybe scheduler should be initialized with non-zero LR?

**Comment by engint-cerebras** [comment]
Closing since scheduler.step should NOT be called before optimizer.step
